app/src/layouts/appLayout/appSidebar/appSidebar.jsx

```diff
@@ -58,6 +59,8 @@ import { ProjectSelector } from '../../common/projectSelector';
   projectRole: activeProjectRoleSelector(state),
   accountRole: userAccountRoleSelector(state),
   extensions: uiExtensionSidebarComponentsSelector(state),
+  organizationSlug: projectOrganizationSlugSelector(state),
+  projectKey: projectKeySelector(state),
```

Comment:
`activeProject` usage need to be removed.

Reply:
Done.